web-BE/routes/api/label/controller.js

```diff
@@ -2,16 +2,20 @@ const labelModel = require('../../../models/label');
 
 const labelController = {
   create: (req, res) => {
-    labelModel.insert();
+    labelModel.insert(req.body);
+    res.json({message:"create a label successfully"})
   },
   read: async (req, res) => {
-    labelModel.select();
+    const labelArr = await labelModel.select();
+    res.json({ labelArray : labelArr })
   },
   update: (req, res) => {
-    labelModel.update();
+    labelModel.update(req.body);
+    res.json({message:"update the label successfully"})
```

Comment:
```suggestion
    res.json({message:"update the label successfully"})
```
띄어쓰기가 가끔 서로 다르게 되어 있는 부분이 있어서 하나로 통일해주면 더 좋을 것 같아요.

Reply:
태그가 조금 이상하게 되었네요 ㅠㅠ.. 이 코드 위에 있는 json 부분을 태그하고 싶었습니다.

Reply:
오 Suggested change 를 그대로 commit suggestion 할 수 있네요,, 승효님 덕분에 깃 기능 하나 더 알아갑니다 ㅋㅋㅋ
말씀해주신 부분은 eslint가 예쁘게 수정해주어서 머지 완료했습니다!!